### PR TITLE
:sparkles: Added bundlesize

### LINKED ISSUE #38: Add bundlesize report in travis
Add https://github.com/siddharthkp/bundlesize on travis.

### PR DESCRIPTION
**Pull request by prateek3255**
Fixes #38

### CODE REVIEW

**Comment by prateek3255** [comment]
Follow these steps to setup bundlesize on travis https://github.com/siddharthkp/bundlesize#build-status-for-github

**Comment by prateek3255** [comment]
Also I am not sure about the sizes so please check this https://github.com/s-yadav/brahmos/pull/40/files#diff-acc29965a79c8e9d28e350efa6e79959 file and update the sizes if any required

File: bundlesize.config.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "files": [
+      {
+        "path": "./dist/brahmos.js",
+        "maxSize": "20 kB"
+      },
+      {
+        "path": "./dist/brahmos.min.js",
+        "maxSize": "10 kB"
```
Comment (s-yadav) [inline bundlesize.config.json]: Keep this as 6.5 kb. Current its 6.0

File: bundlesize.config.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "files": [
+      {
+        "path": "./dist/brahmos.js",
+        "maxSize": "20 kB"
```
Comment (s-yadav) [inline bundlesize.config.json]: Keep this `18 kb`. Current its 16.41

**Comment by s-yadav** [comment]
> I don't think we would need to manually run `test:bundlesize` in `travis.yml`, following these steps [here](https://github.com/siddharthkp/bundlesize#build-status-for-github) would run it automatically on every PR.

I guess you have to add in travis.yml. Otherwise how it will know it has to do bundlesize check.

**Comment by prateek3255** [comment]
Made the bundle size changes and added the test script to travis.
> Otherwise how it will know it has to do bundlesize check.

I thought maybe just by adding the environment variable it would automatically run the test. Anyways don't forget to add the environment variable from here https://github.com/siddharthkp/bundlesize#build-status-for-github for the Github status.

**Pull request merged**